frontend/src/ui/Percentage.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import styled from 'styled-components';
+
+export interface PercentageProps {
+  label: string;
```

Comment:
Any reason you wrote this to accept a string rather than a number representing the percent? I would think users of this component would have the percent they need in a number already rather than a formatted string.

Reply:
The main idea was to take potentially bogus input and clean it up/make it consistent so that percentages are presented consistently in the app both in the table and in the d3/tree map (any other visualizations). So it is not as much about the component (UI) as it is about having shared logic that handles the "cleaning/standardization" of the percentages. The component is just an easy way to make it visible for now.

If we always expect a float, then there's less chance that we'd need to clean it?

If we're putting it on the data source, then perhaps we're expecting the API to clean it up before it presents the json? I'm happy to make any updates/changes that you think make sense (or a separate issue if we think this belongs at the API layer)

Reply:
@alexjball let me know what you think on this and I will adjust etc.

Reply:
Hey @jedpittman, sorry for the late reply. Does the API return a percent string or a float representing a percentage? I think it would be good for the API to provide clean floats, as that's a clearer contract between the API and its clients. 

Since the goal is to share logic, I think `transformNumber` with a float as input might be sufficient for this PR. Both the d3 viz and react table can call it. 

You could also consider adding a jest test that demonstrates using it.

Reply:
no worries. Thanks! I will adjust this request as you describe above.